rtrlib/rtr_mgr.h

```diff
@@ -75,9 +74,11 @@ typedef void (*rtr_mgr_status_fp)(const struct rtr_mgr_group *,
 				  const struct rtr_socket *,
 				  void *);
 
+struct tommy_list_wrapper;
+
 //TODO Add refresh, expire, and retry intervals to config for easier access.
 struct rtr_mgr_config {
-	tommy_list groups;
+	struct tommy_list_wrapper *groups;
```

Comment:
does this need to be a pointer?

Reply:
Yes, because tommy_list_wrapper due to the forward declaration the size of tommy_list_wrapper in rtr_mgr.h.

Reply:
thanks, understood